packages/pyiceberg/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: pyiceberg
+  version: 0.6.0
+  top-level:
+    - hive_metastore
+    - pyiceberg
+    - fb303
+#build:
+#script: python -m pip install . -vv --no-deps --no-build-isolation
+source:
+  url: https://files.pythonhosted.org/packages/01/9e/417eb6ac44c9de920b5c360eb58075553ad6830a7425ea711b2742c3ba25/pyiceberg-0.6.0.tar.gz
+  sha256: 0d965c202a9f49d0daa911c3fe061c55c90fa469bb382ce6440a7799ed35b003
+about:
+  home: https://py.iceberg.apache.org/
+  PyPI: https://pypi.org/project/pyiceberg
+  summary: Apache Iceberg is an open table format for huge analytic datasets
+  license: Apache-2.0
+requirements:
+  host:
```

Comment:
You need to put all these packages in `requirements/run`. They are not used at build time but at runtime.

Reply:
Ah okay, so basically, anything in requirement.txt should go into run and only some special package (say some sort of testing packages) would fall under `requirements/host`?

Reply:
> only some special package

Yes, mostly you don't need to put anything on the `requirements/host`, except for some special cases such as:

1. Numpy: numpy is often used in the build time, you can check the `pyproject.toml` of the project to see if the numpy is defined as a build dependency.
2. Shared libraries: if the package depends on some shared libraries (non python packages), like openssl, openblas, then you need to place it here.